src/aiida/orm/nodes/process/process.py

```diff
@@ -162,6 +162,7 @@ class ProcessNode(Sealable, Node):
     METADATA_INPUTS_KEY: str = 'metadata_inputs'
 
     _unstorable_message = 'only Data, WorkflowNode, CalculationNode or their subclasses can be stored'
+    _logger_adapter = None
```

Comment:
is this a class variable or an instance variable?

Reply:
It is a class variable, but you are right it should be an instance variable. I have updated the code.

Reply:
@khsrali  Ok, it turned out it was a bit more difficult. We cannot initialize the `_logger_adapter` in the constructor, because that path is not followded when loading an existing node from the database. So we _have_ initialize it in the `logger` property and initialize it as `None` as a class variable. Since `logger` sets the adapter on the instance, it won't pollute the class variable though, so it seems fine as a solution.